modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/FDSPropsTransformer/cell_renderers/NameRenderer.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+const NameRenderer = ({
+	itemData,
```

Comment:
Here I'm just curious, where this `itemData` comes from?

Reply:
About this, is it possible to remove the System column in the table? I suppose is needed in `itemData`, so is there a way to hide it?

Reply:
Sounds good! I'll go ahead and remove the System column in the table from StructuresSectionTableFDSView. 👍 

I think `itemData` is one of the props passed into table cell renderers where it contains all the information of that particular line item. It was being used in several examples for frontend-data-set (was looking around [here](https://github.com/liferay/liferay-portal/blob/540d6a1d49322248ffb2591a5ca77e02f8f40103/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/CustomAuthorTableCell.tsx#L23) which really helped in knowing how to apply it!).